support-lambdas/it-test-runner/src/main/scala/com/gu/aws/AwsCloudWatchMetricPut.scala

```diff
@@ -0,0 +1,51 @@
+package com.gu.aws
```

Comment:
this is copied from support-services, as the jar was 56mb with the whole of support-services etc pulled in.
(It's still 25MB even without but we don't want to be too silly at this stage.)
The other option would be being DRY and pulling the aws stuff into a separate library rather than the current tiered setup.

Reply:
Thanks for this comment, I was wondering about it. I think it might be worth extracting this stuff to a separate project as it's not great having copy and paste code around and I think this is not the only place where it has happened, but I'd be happy to leave that out of this PR & put in a health card for it.

Reply:
yes its definitely not the only place.  I think a separate PR.
more generally, if we go down the cohesive modules route rather than layered approach, we could remove a lot of unneeded stuff from the lambda jars and make things more reusable.  if we do decide jar size/startup is a key quality metric we should do more analysis first.

Reply:
in the end, because i copied it into the same package, it's causing some runtime exceptions, as they are all packaged and loaded by the same class loader even though compiled separately.  So I will pull it out in my next PR.